### PR TITLE
Refs #8267 - pass template_url in call to foreman

### PR DESCRIPTION
**Pull request by GregSutcliffe**
Also passes the ?static parameter on correctly.

@witlessbird @dustints could you comment and/or test?

See also: https://github.com/theforeman/foreman/pull/1902

### CODE REVIEW

**Comment by dustints** [comment]
@GregSutcliffe tested with https://github.com/theforeman/foreman/pull/1902 and works for me

File: config/settings.d/templates.yml.example [hunk 1]
```diff
@@ -5,10 +5,14 @@
...
+#
+# :template_url is the URL the host should use to contact the proxy for a template.
 # The default protocol is http on port 80 unless otherwise specified in the url.
 # Examples:
-# smart-proxy.example.com:8080
-# https://smart-proxy.example.com
-# http://1.2.3.4:3000
-#:template_url: https://smart-proxy.example.com:8443
+# http://1.2.3.4:8443
+# https://smart-proxy.example.com # assumes port 80
```
Comment (dmitri-d) [inline config/settings.d/templates.yml.example]: this actually assumes port 443.

File: config/settings.d/templates.yml.example [hunk 2]
```diff
@@ -5,10 +5,14 @@
...
+# smart-proxy.example.com:8080    # assumes http
+#:template_url: http://smart-proxy.example.com:8443
```
Comment (dmitri-d) [inline config/settings.d/templates.yml.example]: I think it would be less confusing to use port 8080 for http.

**Comment by dmitri-d** [comment]
looks good, pls. see my comments re: comments.

File: config/settings.d/templates.yml.example (hunk 1 above)
Comment (GregSutcliffe) [inline config/settings.d/templates.yml.example]: oops, will fix

File: config/settings.d/templates.yml.example (hunk 2 above)
Comment (GregSutcliffe) [inline config/settings.d/templates.yml.example]: 8000 is the default in our settings.yml.example file, maybe i should change both to that?

Comment (dmitri-d) [inline config/settings.d/templates.yml.example]: Sry, that's what I meant.

**Comment by GregSutcliffe** [comment]
@witlessbird updated the comments

**Comment by GregSutcliffe** [comment]
Damn that close button....

**Comment by dmitri-d** [comment]
merged at ab6edc2. Thanks, @GregSutcliffe!

**Pull request closed without merging**